plugins/web/tests/authentication.cpp

```diff
@@ -21,7 +21,7 @@ TEST(token validation) {
   auto serialized_state = state.save();
   REQUIRE_NOERROR(serialized_state);
   vast::plugins::web::authenticator_state recovered_state;
-  recovered_state.initialize_from(*serialized_state);
+  CHECK(!recovered_state.initialize_from(*serialized_state));
```

Comment:
Should this be a `CHECK_EQUAL(..., caf::error{})` instead so you can actually see what error it was in case one is returned?